webpack.config.js

```diff
@@ -28,10 +29,14 @@ module.exports = {
       }
     ]
   },
+  plugins: [
+    new webpack.EnvironmentPlugin(['REACT_APP_SPOTIFY_CLIENT_ID'])
```

Comment:
The `REACT_APP_` part is only needed for app built with `create-react-app`. `SPOTIFY_CLIENT_ID` should suffice.